Header header
string id
int8 number_cameras
sensor_msgs/Image[] data